### PR TITLE
QR code for the address book

### LINKED ISSUE #2929: Address Book: add option to display QR code
There could be a display QR code button for every contact in the address book. 

This would be useful if you want to send Monero from a mobile wallet to a contact that is only available in the GUI wallet.

### PR DESCRIPTION
**Pull request by xmrdsc**
- Show QR code for address book entries like on the receive page
- Introduces new QML component `QRComponent.qml`

fixes #2929

### CODE REVIEW

File: components/QRComponent.qml [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+import QtQuick.Layouts 1.1
+import QtQuick.Dialogs 1.2
+import FontAwesome 1.0
+
+import "../components" as MoneroComponents
+import "../components/effects/" as MoneroEffects
+
+import moneroComponents.Clipboard 1.0
+import moneroComponents.Wallet 1.0
+import moneroComponents.WalletManager 1.0
```
Comment (selsta) [inline components/QRComponent.qml:41]: Line 29 - 46: some of these imports are unused.

**Comment by xmrdsc** [comment]
Thanks, fixed

File: components/QRComponent.qml (hunk 1 above)
Comment (xmrdsc) [reply components/QRComponent.qml]: fixed

**Comment by rating89us** [comment]
In my opinion this page is looking very similar to the receive page now. Maybe some users could mistake the address book with the receive page and give the wrong address to the sender.

**Comment by xmrdsc** [comment]
closed due lack of interest.

**Pull request closed without merging**